presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/ReadFunction.java

```diff
@@ -13,10 +13,23 @@
  */
 package io.prestosql.plugin.jdbc;
 
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
 public interface ReadFunction
 {
     Class<?> getJavaType();
 
+    default boolean isNull(ResultSet resultSet, int columnIndex)
+            throws SQLException
+    {
+        // JDBC is kind of dumb: we need to read the field and then ask
+        // if it was null, which means we are wasting effort here.
+        // We could save the result of the field access if it matters.
+        resultSet.getObject(columnIndex);
+        return resultSet.wasNull();
```

Comment:
Is there any other way to read null value from JDBC, that you heard of?

Reply:
Nope :/